tasks/linters/golangci-lint/0.1/golangci-lint.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: golangci-lint
+spec:
+  description: |
+    This task performs a GolangCI-Lint analysis on shell scripts in a specified folder. It uses the versions of Go and GolangCI-Lint provided through parameters, downloading and configuring them accordingly. Finally, it runs golangci-lint run with the arguments specified in the args parameter.
+  params:
+    - name: context
+      description: path to the directory to use as context
+      default: "."
+    - name: args
+      description: arguments to use for the test command
+      default: -v
+    - name: golint-version
+      description: golangci-lint version to use
+      default: "v1.64.6"
+    - name: go-version
+      description: golangci-lint version to use
+      default: "1.22.9"
```

Comment:
is there a way to make this consistent? I mean, somehow to update this file? Can you check using renovate ?

Reply:
or maybe we can use latest? @psturc wdyt?